common/middleware/countries/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+// npm dependencies
+const lodash = require('lodash')
+
+// Local dependencies
+let countries = require('./data/countries.json')
+const extensions = require('./data/country-record-extension.json')
+
+// Merge the additional data into the register data
```

Comment:
Looks like the comment can be a named method for the function

Reply:
this it a utility rather than middleware. So should probably be in something like `common/utils`

Reply:
Thanks @rauligs and @maxcbc 

@rauligs as discussed we prefer not to declare functions by replacing comments in terms of performance (if not needed)

@maxcbc addressed